101-anf-smb-volume/azuredeploy.json

```diff
@@ -0,0 +1,190 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Same location of resource group for all resources"
+      }
+    },
+    "netappAccountName": {
+      "defaultValue": "[concat('anfacc', uniqueString(resourceGroup().id))]",
+      "type": "String",
+      "metadata": {
+        "description": "Name for the Account. The account name must be unique within the subscription"
+      }
+    },
+    "adUsername": {
+      "type": "string",
+      "metadata": {
+        "description": "The value of Active Directory username"
+      }
+    },
+    "adPassword": {
+      "type": "securestring",
+      "metadata": {
+        "description": "The value of Active Directory password"
+      }
+    },
+    "dnsServerIp": {
+      "type": "string",
+      "metadata": {
+        "description": "DNS server IP address"
+      }
+    },
+    "domainName": {
+      "type": "string",
+      "metadata": {
+        "description": "Domain name"
+      }
+    },
+    "smbServerName": {
+      "type": "string",
+      "metadata": {
+        "description": "SMB server name"
+      }
+    },
+    "netAppPoolName": {
+      "defaultValue": "[concat('pool', uniqueString(resourceGroup().id))]",
+      "type": "String",
+      "metadata": {
+        "description": "Name for the capacity pool. The capacity pool name must be unique for each NetApp account."
+      }
+    },
+    "poolSizeBytes": {
+      "type": "int",
+      "defaultValue": 4398046511104,
+      "minValue": 4398046511104,
+      "maxValue": 549755813888000,
+      "metadata": {
+        "description": "Size of the capacity pool. The minimum  size is 4 TiB."
+      }
+    },
+    "netAppVolumeName": {
+      "defaultValue": "[concat('volume', uniqueString(resourceGroup().id))]",
+      "type": "String",
+      "metadata": {
+        "description": "Name for the Volume. A volume name must be unique within each capacity pool. It must be at aleast three characters long and you can use any alphanumeric characters."
+      }
+    },
+    "volSizeBytes": {
+      "type": "int",
+      "defaultValue": 107374182400,
+      "minValue": 107374182400,
+      "maxValue": 109951162777600,
+      "metadata": {
+        "description": "Amount of logical storage that is allocated to the volume."
+      }
+    },
+    "virtualNetworkName": {
+      "type": "String",
+      "metadata": {
+        "description": "The name of the VNET to add a subnet to."
+      }
+    },
+    "subnetName": {
```

Comment:
lets rename this to something like anfSubnetName, so it gets clear that it is the subnet where the volume will be created. Similarly, the beginning of the description must follow. E.g. The name of the subnet where the ANF volume will be created. This subnet will be delegated to Microsoft.NetApp/volumes.